integration/kubernetes/e2e_conformance/run.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This script runs the Sonobuoy e2e Conformance tests.
+# Run this script once your K8s cluster is running.
+# WARNING: it is prefered to use containerd as the 
+# runtime interface instead of cri-o as we have seen
+# errors with cri-o that still need to be debugged.
+
+set -e
+
+export KUBECONFIG=$HOME/.kube/config
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../../../lib/common.bash"
+
+RUNTIME="${RUNTIME:-kata-runtime}"
+
+# Check if Sonobuoy is still running every 5 minutes.
+WAIT_TIME=300
+
+# Add a global timeout of 2 hours to stop the execution
+# in case Sonobuoy gets hanged.
+GLOBAL_TIMEOUT=$((WAIT_TIME*24))
+
+create_kata_webhook() {
+	pushd "${SCRIPT_PATH}/../../../kata-webhook"
+	# Create certificates for the kata webhook
+	./create-certs.sh
+
+	# Apply kata-webhook deployment
+	kubectl apply -f deploy/
+	popd
+}
+
+run_sonobuoy() {
+	# Get Sonobuoy repo
+	sonobuoy_repo="github.com/heptio/sonobuoy"
+	go get -u "$sonobuoy_repo"
+
+	# Run Sonobuoy e2e tests
+	sonobuoy run
+	info "When using kata as k8s runtime, the tests take around 2 hrs to finish."
+
+	start_time=$(date +%s)
+	estimated_end_time=$((start_time + GLOBAL_TIMEOUT))
+
+	# Wait for the sonobuoy pod to be running.
+	kubectl wait --for condition=Ready pod sonobuoy -n heptio-sonobuoy
+
+	while sonobuoy status | grep -Eq "running|pending" && [ "$(date +%s)" -le "$estimated_end_time" ]; do
+		info "sonobuoy still running, sleeping $WAIT_TIME seconds"
+		sleep "$WAIT_TIME"
+	done
+
+	# Retrieve results
+	e2e_result_dir="$(mktemp -d kata_e2e_results)"
```

Comment:
This is not missing `X` ?: mktemp -d kata_e2e_results.XXXXX

Reply:
should we remove this after the end of this test?

Reply:
it would be under /tmp/ and I prefer to not remove this as the user might want to check the results directory after the tests are executed.

Reply:
@chavafg thanks for the explanation :)